apps/wappush/js/cp_screen_helper.js

```diff
@@ -455,7 +452,7 @@ var CpScreenHelper = (function() {
                                                      authInfo);
       } catch (e) {
         message = finishConfirmDialog.querySelector('strong');
-        message.textContent = _(e.message);
+        navigator.mozL10n.setAttributes(message, e.message);
```

Comment:
You don't follow the guideline from https://developer.mozilla.org/en-US/Firefox_OS/Developing_Gaia/localization_code_best_practices#Writing_code_that_iterates_over_many_l10n_strings

I don't know if this is really an issue here though -- it's still quite painful to go in auth.js and look at the possible values.